app/src/main/java/com/example/githubusers/presenter/GithubUsersPresenter.java

```diff
@@ -0,0 +1,52 @@
+package com.example.githubusers.presenter;
+
+import android.util.Log;
+
+import com.example.githubusers.model.GithubUsers;
+import com.example.githubusers.model.GithubUsersResponse;
+import com.example.githubusers.service.GithubService;
+import com.example.githubusers.view.UsersParentView;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+
+public class GithubUsersPresenter {
+    private static final String TAG = "GithubUsersPresenter";
+    private UsersParentView usersView;
+    private GithubService githubService;
+
+    public GithubUsersPresenter(UsersParentView view) {
+        this.usersView = view;
+
+        if (this.githubService == null) {
+            this.githubService = new GithubService();
```

Comment:
Good Job 👍 @leonardnjura 
However, I will suggest you pass in the `service` class as an argument to the presenter's constructor. This is to prevent tight coupling of the MVP components and make the presenter **unit testable**

Reply:
On it